src/atoms/Flex/__tests__/Flex.native.test.js

```diff
@@ -1,15 +1,140 @@
 import React from 'react';
 import { render } from '@testing-library/react-native';
-import { View } from 'react-native';
+
+import View from '../../View';
 import Flex from '../index';
+import Space from '../../Space';
+import Size from '../../Size';
+
+const defaultFlexStyles = { flexDirection: 'column' };
+const defaultSpaceStyles = {};
+const defaultSizeStyles = {};
 
 describe('Renders <Flex /> correctly', () => {
-  it('snapshot testing', () => {
-    const { container } = render(
+  it('renders child of <Flex />(no styles) correctly', () => {
+    const { container, getByTestId } = render(
       <Flex>
-        <View />
+        <View testID={'flex-view'} />
+      </Flex>,
+    );
+    const childView = getByTestId('flex-view');
+    expect(childView.props.style).toMatchObject([defaultFlexStyles]);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('<Flex /> and <Space /> together plays properly', () => {
+  it('renders child of <Flex />(no styles) and <Space />(no styles) correctly', () => {
+    const { container, getByTestId } = render(
+      <Flex>
+        <Space>
+          <View testID={'flex-space-view'} />
+        </Space>
+      </Flex>,
+    );
+    const childView = getByTestId('flex-space-view');
+    expect(childView.props.style).toMatchObject([defaultSpaceStyles, defaultFlexStyles]);
+    expect(container).toMatchSnapshot();
+  });
+
+  it('renders child of <Space />(no styles) and <Flex />(no styles) correctly', () => {
+    const { container, getByTestId } = render(
+      <Space>
+        <Flex>
+          <View testID={'space-flex-view'} />
+        </Flex>
+      </Space>,
+    );
+    const childView = getByTestId('space-flex-view');
+    expect(childView.props.style).toMatchObject([defaultFlexStyles, defaultSpaceStyles]);
+    expect(container).toMatchSnapshot();
+  });
+});
+
+describe('<Flex /> and <Size /> together plays properly', () => {
+  it('renders child of <Flex />(no styles) and <Size />(no styles) correctly', () => {
```

Comment:
```suggestion
  it('renders <Size /> as child component of <Flex />', () => {
```

Reply:
Not just Flex as child of Size but it mainly tests the View which is a child of Flex which in turn is a child of Size has received correct styles.